shell/client/shell.html

```diff
@@ -166,13 +167,27 @@
         {{else}}
           {{#if selectedTab.myFiles}}
             <div class="button-box">
-              <button id="install-apps-button">Install apps</button>
-              <button id="upload-app-button">Upload an app</button>
-              <button id="restore-backup-button">Restore a backup</button>
+              <button id="install-apps-button" class="{{#if overQuota}}over-quota{{/if}}">
+                  Install apps</button>
+              <button id="upload-app-button" class="{{#if overQuota}}over-quota{{/if}}">
+                  Upload an app</button>
+              <button id="restore-backup-button" class="{{#if overQuota}}over-quota{{/if}}">
+                  Restore a backup</button>
             </div>
           {{/if}}
         {{/if}}
 
+        {{#unless selectedTab.sharedWithMe}}
+          {{#if quotaEnabled}}
+            <div id="storage-usage" class="{{#if overQuota}}over-quota{{/if}}">{{storageUsage}}
```

Comment:
This shows global storage beneath each app, which is a bit unintuitive. I definitely understand that we want users to be constantly aware of how close they are to the limit, but this text appears in a seemingly app specific context, and could be easily confused to read the space usage of only this app's grains.

This may have to wait on the shell re-design to fix, since I can't think of a better place to put it for now.